.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,10 +11,10 @@ assignees: mrjesen
 A clear and concise description of what the bug is.
 
 **Client Software**
-HMCL (MC: 1.16.1)
+HMCL (MC: 1.16.5)
```

Comment:
this is just a sample,

Reply:
I updated this part since 1.16.5 is newest.